tensorflow_datasets/testing/dataset_builder_testing.py

```diff
@@ -137,7 +137,10 @@ class DatasetBuilderTestCase(parameterized.TestCase, test_utils.SubTestCase):
   EXAMPLE_DIR = None
   OVERLAPPING_SPLITS = []
   MOCK_OUT_FORBIDDEN_OS_FUNCTIONS = True
-
+  SKIP_CHECKSUMS = False
+  URLS = []
```

Comment:
All class will share the same urls, which might create conflicts between multiple instances. This shouldn't be a global attribute.
I would move it `setUpClass` instead.
Also UPPER_CASE mean constant in Python, I would rename to `_download_urls`.